boa/src/vm/mod.rs

```diff
@@ -37,8 +37,167 @@ impl<'a> VM<'a> {
 
                     self.stack.push(val);
                 }
+                Instruction::Sub => {
+                    let r = self.stack.pop().unwrap();
+                    let l = self.stack.pop().unwrap();
```

Comment:
We have a lot of unwraps!
I wonder if we should make a self.pop method that unwraps then returns he value or panics with the correct message, then call that.

If we ever move to an array we then only need to change that method.